lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -294,6 +294,7 @@ function New-XmlResult
 }
 
 Add-Type -TypeDefinition $hostSource -Language CSharp
+$global:DefaultWorkingDirectory = (Get-Location).Path
```

Comment:
This will produce bugs with difference between .NET's idea of cwd and PowerShell's. Can't remember link off top of head, but we have to specify filesystem provider. Looking now